update.go

```diff
@@ -19,6 +20,17 @@ eiFH+GffIMk09RFqkcMh5rIPu3ykm5V8
 `)
 
 func update(channel string, silent bool) error {
+	now := time.Now()
+	equinoxEOL, _ := time.Parse(time.RFC3339, "2021-09-30T00:00:00Z")
```

Comment:
Please manually check this works with a time in the past, it'd suck to release this and have it blow up once we're past the `equinoxEOL`

Reply:
I manually checked the parsing / time comparison code in the Go playground. Returning `nil` is copied from further down: https://github.com/rainforestapp/rainforest-cli/pull/409/files#diff-93ae3e89134c5b7d0b37869c5b950405c369d60559b300ed579b85b9f89f3884L40-L46

That said, I got a reply from Equinox overnight, looks like the plan is for their servers to respond with "no update available" responses (so as to not break existing clients), so there's not much urgency for this PR to go out.